setup.py

```diff
@@ -12,7 +12,7 @@
     py_modules=['pytest_codestyle'],
     python_requires='>=3.6,<4',
     install_requires=[
-        'pytest>=3.3,<3.4',
+        'pytest>=3.0,<3.4',
```

Comment:
Please also change `<3.4` to `<4` :)
I assume that pytest follow [Semantic Versioning](https://semver.org/).

Reply:
Done